webapp/src/components/expanded_view/component.tsx

```diff
@@ -702,6 +728,24 @@ export default class ExpandedView extends React.PureComponent<Props, State> {
                                 }
                                 unavailable={noInputDevices || noAudioPermissions}
                             />
+                            {globalRhsSupported && (
+                                <ControlsButton
+                                    id='calls-popout-chat-button'
+                                    onToggle={this.toggleChat}
+                                    tooltipText={chatToolTipText}
+                                    tooltipSubtext={chatToolTipSubtext}
+                                    // eslint-disable-next-line no-undefined
+                                    shortcut={undefined}
```

Comment:
@tanmay-des Would it make sense to have a keyboard shortcut to toggle the chat thread?

Reply:
Suggestion: `CTRL/CMD + .` (existing and already works if call-thread opened at least once, but extended to better handle in expanded-view/expanded window)

![CleanShot 2022-10-26 at 18 57 54](https://user-images.githubusercontent.com/11724372/198160546-e4f041ca-15ae-4862-b862-15291d6a74e0.gif)